recipes/diffpy.srfit/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "diffpy.srfit" %}
+{% set version = "3.1.0rc0" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
thanks for the link.  Very helpful.

Reply:
It is not completely clear to me how we do this labelling in the staged-recipe.   To make things easier I think we could wait for our full release to appear on PyPI and then change this to a non-rc release.   I guess merging to an `rc` branch as described in the article by Eric is possible from the feedstock but not hte staged recipe?  Anyway, this is easily fixed if we skip the rc release to conda-forge this time.

however, please see below for the licensing issue.....

Reply:
I think this is better also.  We will work on that and loop back when it is done.  

We wanted to do it this way as we wanted to conda install this into another package and wanted to test everything including the conda imports before it is public, but since noone is using it as it doesn't exist on CF yet, I think it is just easier to push up a full release.

Reply:
sorry, that was ambigous I see when I reread, what I mean is that we will make a stable release instead of the rc